### PR TITLE
Milestone 2 -Interactive list

### PR DESCRIPTION
**Pull request by Jmagero**

### CODE REVIEW

**Comment by Jmagero** [comment]
- [x] Add methods related to drag / drop and sorting functionality.
- [x] Add methods related to the status updates (completed: true / false).
- [x] Add event listeners to list items (dragstart, dragover, drop) and to checkbox (change).
- [x] Updated items object's values for index and completed keys upon user actions.
- [x] Add methods to store the updated array of items in local storage

**Comment by bellom** [comment]
> * [x]  Add methods related to drag / drop and sorting functionality.
> * [x]  Add methods related to the status updates (completed: true / false).
> * [x]  Add event listeners to list items (dragstart, dragover, drop) and to checkbox (change).
> * [x]  Updated items object's values for index and completed keys upon user actions.
> * [x]  Add methods to store the updated array of items in local storage

Good job adding a PR description but you can always edit the first one with no PR description.

File: dist/index.html [hunk 1]
```diff
@@ -1 +1,22 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+    <title>To-dos-List</title>
+<script defer src="main.js"></script></head>
+<body>
+    <main class="m-4 w-70">
+        <div class="border border-secondary d-flex flex-column align-items-center justify-content-center rounded-3">
+            <h2 class="py-1 px-2 mx-2">To-Dos</h2>
+              <ul id="tasks" class="listClass d-flex w-50 flex-column" class = "drag-sort-enable"></ul>
```
Comment (bellom) [inline dist/index.html]: Let's try to follow good [HTML/CSS practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
`class = "drag-sort-enable"` Use space-less equal signs, like this `class="drag-sort-enable"`

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Jmagero  👏 , 

### Awesome job so far!! :+1: 

# Status: Required Changes! ♻️ 

1. Kindly check the comments under the review.
2. Kindly implement the drag feature and let's work a little on the UI to fit in with the requirement.


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Jmagero** [comment]
- [x]  Implement the drag feature and let's work a little on the UI to fit in with the requirement

**Review by ZawadiSumba66 — APPROVED** [review]
#### Hi @Jmagero 👋 , 
### STATUS:APPROVED ✔️ 
Good job implementing the changes requested by the previous reviewer 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**